verticapy/tests/vModel/test_normalizer.py

```diff
@@ -0,0 +1,231 @@
+                model2.name
+            )
+        )
+        prediction = model2.cursor.fetchone()[0]
+        assert prediction == pytest.approx(md.transform([[3.0, 11.0, 93.0]])[0][0])
+        # Robust Zscore
+        model3 = Normalizer("norm_model_test2", cursor=model.cursor, method = "robust_zscore")
+        model3.drop()
+        model3.fit(
+            "public.winequality", ["citric_acid", "residual_sugar", "alcohol"]
+        )
+        md = model3.to_sklearn()
+        model3.cursor.execute(
+            "SELECT APPLY_NORMALIZE(citric_acid, residual_sugar, alcohol USING PARAMETERS model_name = '{}', match_by_pos=True) FROM (SELECT 3.0 AS citric_acid, 11.0 AS residual_sugar, 93. AS alcohol) x".format(
+                model3.name
+            )
+        )
+        prediction = model3.cursor.fetchone()[0]
+        assert prediction == pytest.approx(md.transform([[3.0, 11.0, 93.0]])[0][0])
+
+    def test_get_transform(self, winequality_vd, model):
+        # Zscore
+        winequality_trans = model.transform(
+            winequality_vd,
+            X=["citric_acid", "residual_sugar", "alcohol"]
+        )
+        assert winequality_trans["citric_acid"].mean() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        assert winequality_trans["residual_sugar"].mean() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        assert winequality_trans["alcohol"].mean() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        # Minmax
+        model2 = Normalizer("norm_model_test2", cursor=model.cursor, method = "minmax")
+        model2.drop()
+        model2.fit(
+            "public.winequality", ["citric_acid", "residual_sugar", "alcohol"]
+        )
+        winequality_trans = model2.transform(
+            winequality_vd,
+            X=["citric_acid", "residual_sugar", "alcohol"]
+        )
+        assert winequality_trans["citric_acid"].min() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        assert winequality_trans["residual_sugar"].max() == pytest.approx(
+            1.0, abs=1e-6
+        )
+        assert winequality_trans["alcohol"].min() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        # Robust Zscore
+        model3 = Normalizer("norm_model_test2", cursor=model.cursor, method = "robust_zscore")
+        model3.drop()
+        model3.fit(
+            "public.winequality", ["citric_acid", "residual_sugar", "alcohol"]
+        )
+        winequality_trans = model3.transform(
+            winequality_vd,
+            X=["citric_acid", "residual_sugar", "alcohol"]
+        )
+        assert winequality_trans["citric_acid"].median() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        assert winequality_trans["residual_sugar"].median() == pytest.approx(
+            0.0, abs=1e-6
+        )
+        assert winequality_trans["alcohol"].median() == pytest.approx(
+            0.0, abs=1e-6
+        )
+
```

Comment:
Please add `model3.drop()` here to clean up the database.

Reply:
Done;